src/pages/top.js

```diff
@@ -192,26 +209,36 @@ function Top({ trees, planters, countries, organizations }) {
 }
 
 export async function getServerSideProps() {
-  const [trees, countries, planters, organizations] = await Promise.all([
-    getFeaturedTrees(), //
-    getCountryLeaderboard(),
-    (async () => {
-      const data = await utils.requestAPI('/planters/featured');
-      log.warn('planters', data);
-      return data.planters;
-    })(),
-    (async () => {
-      const data = await utils.requestAPI('/organizations/featured');
-      log.warn('organizations', data);
-      return data.organizations;
-    })(),
-  ]);
+  const [trees, countries, planters, organizations, tokens] = await Promise.all(
+    [
+      getFeaturedTrees(), //
+      getCountryLeaderboard(),
+      (async () => {
+        const data = await utils.requestAPI('/planters/featured');
+        log.warn('planters', data);
+        return data.planters;
+      })(),
+      (async () => {
+        const data = await utils.requestAPI('/organizations/featured');
+        log.warn('organizations', data);
+        return data.organizations;
+      })(),
+      (async () => {
+        const data = await utils.requestAPI(
+          '/tokens?wallet=ba95a148-7ff8-42bb-8028-99b747b86ae1',
+        );
+        log.warn('wallets', data);
+        return data.tokens;
```

Comment:
This is wrong, this return tokens, we need wallets (the top page will show wallets picked by us in advance, so called `featured wallet`)

So you should propobly go: `https://dev-k8s.treetracker.org/query/wallets?limit=10`

I checked, seems this API is missing, I opened an issue for this:

https://github.com/Greenstand/treetracker-query-api/issues/146

Reply:
I figured it was wrong, but it's the only thing I could find. How do I see a list of all the apis available? I couldn't figure that out.